src/main/java/de/fraunhofer/aisec/cpg/graph/Statement.java

```diff
@@ -38,7 +38,8 @@
    * VariableDeclaration} extracted from Block because for, while, if, switch can declare locals in
    * their condition or initializers
    */
-  private @SubGraph("AST") List<VariableDeclaration> locals = new ArrayList<>();
+  @SubGraph("AST")
```

Comment:
That's what I removed in my new PR, maybe that's something that can already be removed in this one

Reply:
can do

Reply:
See discussion on Mattermost, I have decided against removing it, since we are not quite clear if it is actually an AST node if just a `Statement` is used as part of the condition or initialiser of a `for` or `if`.